src/App.tsx

```diff
@@ -1,24 +1,188 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Header } from './components/Heder/Heder';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorNotification }
+  from './components/ErrorNotification/ErrorNotification';
+import { Todo } from './types/Todo';
+import {
+  createTodo, deleteTodo, getTodo, updateTodo,
+} from './api/todos';
+import { FilterTodos } from './enum/FilterTodos';
+import { TodoError } from './enum/TodoError/TodoError';
 
-const USER_ID = 0;
+const USER_ID = 12111;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [chooseFilter, setChooseFilter] = useState(FilterTodos.All);
+  const [errorMessage, setErrorMesage] = useState<string | null>('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoader, setIsLoader] = useState(false);
+  const [queryInput, setQueryInput] = useState('');
+  const [arrayLoader, setArrayLoader] = useState<number[] | null>([]);
+
+  useEffect(() => {
+    getTodo(USER_ID).then(setTodos)
+      .catch(() => setErrorMesage(TodoError.UnableLoad));
+  }, []);
+
+  const filterTodos = useMemo(() => todos.filter((todo: Todo) => {
+    switch (chooseFilter) {
+      case FilterTodos.Active:
+        return !todo.completed;
+
+      case FilterTodos.Completed:
+        return todo.completed;
+
+      default:
+        return todo;
+    }
```

Comment:
No need to call filter() in case FilterTodos.All
```suggestion
  const filterTodos = useMemo(() => {
    switch (chooseFilter) {
      case FilterTodos.Active:
        return todos.filter((todo: Todo) =>!todo.completed);

      case FilterTodos.Completed:
        return  todos.filter((todo: Todo) =>todo.completed);

      default:
        return todos;
    }
```